src/NumberController.js

```diff
@@ -197,14 +203,15 @@ export default class NumberController extends Controller {
 		const onBlur = () => {
 			this._inputFocused = false;
 			this.updateDisplay();
+			this._callOnFinishChange();
 		};
 
```

Comment:
`_initInput` and `_initSlider` both have all their event listeners bound at the end of the method for consistency.